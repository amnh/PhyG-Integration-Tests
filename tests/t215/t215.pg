--fuse(none), inputting tcm file (dayhoff) with amino acids -- update: had to remove dayhoff tcms to speed up run
--originally fuse(nni)
set(seed:1634561640)
read(prefasta:"metazoa-aa-1.fas")
read(prefasta:"metazoa-aa-2.fas")
read(newick:"../t85/t85.tre")
fuse(none, pairs:3)
report("t215.csv", data, overwrite)
report("t215.tre", newick, graphs, overwrite)