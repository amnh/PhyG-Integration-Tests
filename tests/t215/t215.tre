(Antarctic_neomeniomorph:26.0,(Lingula_anatina:0.0,(Cerebratulus_lacteus:20.0,(Euprymna_scolopes:11.0,(Mytilus_californianus:56.0,(Perotrochus_lucaya:8.0,((Nautilus:18.0,(Nucula:13.0,(Octopus:42.0,(Solemya:29.0,((Greenland_neomeniomorph:0.0,Ilyanassa_obsoleta:0.0)HTU39:0.0,(Helobdella_robusta:14.0,(Lottia_gigantea:11.0,Scaphopods:80.0)HTU38:0.0)HTU40:2.0)HTU41:9.0)HTU42:4.0)HTU43:9.0)HTU44:1.0)HTU45:0.0,(Lymnaea_stagnalis:8.0,((Venerupis_decussatus:2.0,(Chaetopleura_apiculata:0.0,Yoldia_limatula:0.0)HTU34:0.0)HTU35:1.0,(Littorina_littorea:3.0,(Venerupis_philippinarum:8.0,(Aplysia_californica:4.0,(Crassostrea_virginica:15.0,Siphonaria_pectinata:23.0)HTU31:0.0)HTU32:1.0)HTU33:0.0)HTU36:2.0)HTU37:0.0)HTU46:1.0)HTU29:1.0)HTU30:5.0)HTU25:4.0)HTU26:2.0)HTU27:0.0)HTU28:27.0)HTU24[1381.0];
