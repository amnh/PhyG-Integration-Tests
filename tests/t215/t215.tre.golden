(Antarctic_neomeniomorph:0.0,(Lingula_anatina:0.0,((Cerebratulus_lacteus:52.0,Euprymna_scolopes:21.0)HTU40:10.0,(Mytilus_californianus:47.0,(((Lottia_gigantea:31.0,(Scaphopods:191.0,(Nautilus:15.0,(Yoldia_limatula:9.0,(Lymnaea_stagnalis:6.0,(Chaetopleura_apiculata:15.0,((Aplysia_californica:54.0,Littorina_littorea:35.0)HTU29:38.0,(Crassostrea_virginica:40.0,(Perotrochus_lucaya:3.0,Siphonaria_pectinata:30.0)HTU46:0.0)HTU44:0.0)HTU45:13.0)HTU30:0.0)HTU31:0.0)HTU32:4.0)HTU33:6.0)HTU34:21.0)HTU35:3.0,(Helobdella_robusta:14.0,(Greenland_neomeniomorph:75.0,((Ilyanassa_obsoleta:0.0,Solemya:0.0)HTU25:0.0,(Nucula:14.0,Octopus:59.0)HTU26:1.0)HTU27:3.0)HTU28:0.0)HTU36:0.0)HTU37:22.0,(Venerupis_decussatus:31.0,Venerupis_philippinarum:1.0)HTU38:3.0)HTU39:0.0)HTU41:43.0)HTU42:0.0)HTU43:0.0)HTU24[1431.0];
(Antarctic_neomeniomorph:0.0,(Lingula_anatina:0.0,((Cerebratulus_lacteus:51.0,Euprymna_scolopes:23.0)HTU26:10.0,((Venerupis_decussatus:31.0,Venerupis_philippinarum:3.0)HTU25:1.0,(Mytilus_californianus:49.0,(Greenland_neomeniomorph:69.0,((Ilyanassa_obsoleta:0.0,(Nucula:16.0,Solemya:15.0)HTU32:0.0)HTU33:0.0,(Lottia_gigantea:30.0,(Octopus:41.0,(Nautilus:9.0,(((Lymnaea_stagnalis:6.0,(Yoldia_limatula:11.0,(Chaetopleura_apiculata:17.0,(Aplysia_californica:54.0,Littorina_littorea:34.0)HTU38:33.0)HTU39:0.0)HTU40:1.0)HTU41:0.0,(Perotrochus_lucaya:2.0,(Crassostrea_virginica:15.0,Siphonaria_pectinata:7.0)HTU37:1.0)HTU42:0.0)HTU43:16.0,(Helobdella_robusta:12.0,Scaphopods:6.0)HTU45:46.0)HTU46:8.0)HTU44:17.0)HTU36:9.0)HTU35:4.0)HTU34:3.0)HTU30:11.0)HTU31:2.0)HTU27:14.0)HTU28:0.0)HTU29:0.0)HTU24[1418.0];
(Antarctic_neomeniomorph:0.0,(Lingula_anatina:0.0,((Cerebratulus_lacteus:54.0,Euprymna_scolopes:20.0)HTU26:10.0,((Venerupis_decussatus:31.0,Venerupis_philippinarum:3.0)HTU25:1.0,(Mytilus_californianus:49.0,((Lottia_gigantea:28.0,(Greenland_neomeniomorph:73.0,(Ilyanassa_obsoleta:0.0,(Nucula:17.0,Solemya:14.0)HTU32:0.0)HTU33:0.0)HTU34:5.0)HTU35:3.0,(Octopus:43.0,(Nautilus:12.0,(((Lymnaea_stagnalis:6.0,(Yoldia_limatula:11.0,(Chaetopleura_apiculata:17.0,(Aplysia_californica:54.0,Littorina_littorea:34.0)HTU38:33.0)HTU39:0.0)HTU40:1.0)HTU41:0.0,(Perotrochus_lucaya:2.0,(Crassostrea_virginica:15.0,Siphonaria_pectinata:7.0)HTU37:1.0)HTU42:0.0)HTU43:16.0,(Helobdella_robusta:12.0,Scaphopods:6.0)HTU45:46.0)HTU46:8.0)HTU44:19.0)HTU36:5.0)HTU30:23.0)HTU31:3.0)HTU27:14.0)HTU28:0.0)HTU29:0.0)HTU24[1418.0];
(Antarctic_neomeniomorph:0.0,(Lingula_anatina:0.0,((Cerebratulus_lacteus:52.0,Euprymna_scolopes:25.0)HTU35:9.0,(Mytilus_californianus:45.0,((Helobdella_robusta:9.0,(Lottia_gigantea:33.0,(Ilyanassa_obsoleta:2.0,((Greenland_neomeniomorph:74.0,Solemya:15.0)HTU27:4.0,(Nucula:21.0,(Octopus:45.0,(Nautilus:9.0,(Scaphopods:193.0,(Chaetopleura_apiculata:16.0,((Aplysia_californica:54.0,Littorina_littorea:34.0)HTU42:38.0,(Yoldia_limatula:11.0,(Lymnaea_stagnalis:6.0,(Perotrochus_lucaya:3.0,(Crassostrea_virginica:16.0,Siphonaria_pectinata:7.0)HTU41:1.0)HTU46:0.0)HTU45:0.0)HTU44:0.0)HTU43:14.0)HTU39:6.0)HTU40:9.0)HTU25:18.0)HTU26:8.0)HTU28:0.0)HTU29:1.0)HTU30:0.0)HTU31:0.0)HTU32:18.0,(Venerupis_decussatus:31.0,Venerupis_philippinarum:1.0)HTU33:3.0)HTU34:0.0)HTU36:42.0)HTU37:0.0)HTU38:0.0)HTU24[1418.0];
(Antarctic_neomeniomorph:0.0,(Lingula_anatina:0.0,((Cerebratulus_lacteus:54.0,Euprymna_scolopes:23.0)HTU26:10.0,((Mytilus_californianus:9.0,(Venerupis_decussatus:32.0,Venerupis_philippinarum:1.0)HTU25:3.0)HTU27:12.0,(Nucula:25.0,((Octopus:46.0,(Nautilus:11.0,(Scaphopods:194.0,(Chaetopleura_apiculata:16.0,((Aplysia_californica:54.0,Littorina_littorea:34.0)HTU35:38.0,(Yoldia_limatula:11.0,(Lymnaea_stagnalis:6.0,(Perotrochus_lucaya:3.0,(Crassostrea_virginica:16.0,Siphonaria_pectinata:7.0)HTU32:1.0)HTU33:0.0)HTU34:0.0)HTU36:0.0)HTU37:14.0)HTU38:6.0)HTU39:8.0)HTU40:20.0)HTU41:10.0,((Greenland_neomeniomorph:72.0,Solemya:16.0)HTU42:4.0,(Ilyanassa_obsoleta:2.0,(Helobdella_robusta:12.0,Lottia_gigantea:2.0)HTU46:0.0)HTU45:1.0)HTU44:2.0)HTU43:2.0)HTU30:13.0)HTU31:44.0)HTU28:0.0)HTU29:0.0)HTU24[1415.0];
(Antarctic_neomeniomorph:0.0,(Lingula_anatina:0.0,((Cerebratulus_lacteus:55.0,Euprymna_scolopes:20.0)HTU26:10.0,((Mytilus_californianus:9.0,(Venerupis_decussatus:32.0,Venerupis_philippinarum:1.0)HTU25:3.0)HTU27:12.0,((Octopus:46.0,(Nautilus:10.0,(Scaphopods:193.0,(Chaetopleura_apiculata:16.0,((Aplysia_californica:54.0,Littorina_littorea:34.0)HTU35:38.0,(Yoldia_limatula:11.0,(Lymnaea_stagnalis:6.0,(Perotrochus_lucaya:3.0,(Crassostrea_virginica:16.0,Siphonaria_pectinata:7.0)HTU32:1.0)HTU33:0.0)HTU34:0.0)HTU36:0.0)HTU37:13.0)HTU38:6.0)HTU39:8.0)HTU40:20.0)HTU41:7.0,(Nucula:21.0,((Greenland_neomeniomorph:71.0,Solemya:16.0)HTU42:4.0,(Ilyanassa_obsoleta:2.0,(Helobdella_robusta:12.0,Lottia_gigantea:2.0)HTU46:0.0)HTU45:1.0)HTU44:1.0)HTU43:2.0)HTU30:24.0)HTU31:44.0)HTU28:0.0)HTU29:0.0)HTU24[1415.0];
(Antarctic_neomeniomorph:0.0,(Lingula_anatina:0.0,((Cerebratulus_lacteus:54.0,Euprymna_scolopes:20.0)HTU40:10.0,((Venerupis_decussatus:31.0,Venerupis_philippinarum:3.0)HTU38:1.0,(Mytilus_californianus:49.0,((Lottia_gigantea:28.0,(Greenland_neomeniomorph:73.0,(Ilyanassa_obsoleta:0.0,(Nucula:17.0,Solemya:14.0)HTU32:0.0)HTU33:0.0)HTU34:5.0)HTU35:3.0,(Octopus:44.0,(Nautilus:13.0,((Lymnaea_stagnalis:6.0,(Yoldia_limatula:12.0,(Chaetopleura_apiculata:16.0,((Aplysia_californica:54.0,Littorina_littorea:35.0)HTU25:38.0,(Crassostrea_virginica:40.0,(Perotrochus_lucaya:3.0,Siphonaria_pectinata:30.0)HTU46:0.0)HTU44:0.0)HTU45:13.0)HTU26:0.0)HTU27:0.0)HTU28:6.0,(Helobdella_robusta:12.0,Scaphopods:6.0)HTU29:46.0)HTU30:8.0)HTU31:18.0)HTU36:5.0)HTU37:23.0)HTU39:3.0)HTU41:14.0)HTU42:0.0)HTU43:0.0)HTU24[1406.0];
(Antarctic_neomeniomorph:0.0,(Lingula_anatina:0.0,((Cerebratulus_lacteus:54.0,Euprymna_scolopes:20.0)HTU40:10.0,((Venerupis_decussatus:31.0,Venerupis_philippinarum:3.0)HTU38:1.0,(Mytilus_californianus:49.0,((Lottia_gigantea:28.0,(Greenland_neomeniomorph:73.0,(Ilyanassa_obsoleta:0.0,(Nucula:17.0,Solemya:14.0)HTU32:0.0)HTU33:0.0)HTU34:5.0)HTU35:3.0,(Octopus:44.0,(Nautilus:13.0,((Lymnaea_stagnalis:6.0,(Yoldia_limatula:12.0,(Chaetopleura_apiculata:16.0,((Aplysia_californica:54.0,Littorina_littorea:35.0)HTU25:38.0,(Siphonaria_pectinata:30.0,(Crassostrea_virginica:38.0,Perotrochus_lucaya:2.0)HTU46:0.0)HTU44:0.0)HTU45:13.0)HTU26:0.0)HTU27:0.0)HTU28:6.0,(Helobdella_robusta:12.0,Scaphopods:6.0)HTU29:46.0)HTU30:8.0)HTU31:18.0)HTU36:5.0)HTU37:23.0)HTU39:3.0)HTU41:14.0)HTU42:0.0)HTU43:0.0)HTU24[1406.0];
(Antarctic_neomeniomorph:0.0,(Lingula_anatina:0.0,((Cerebratulus_lacteus:54.0,Euprymna_scolopes:20.0)HTU35:10.0,((Venerupis_decussatus:31.0,Venerupis_philippinarum:3.0)HTU33:1.0,(Mytilus_californianus:49.0,((Lottia_gigantea:28.0,(Greenland_neomeniomorph:73.0,(Ilyanassa_obsoleta:0.0,(Nucula:17.0,Solemya:14.0)HTU27:0.0)HTU28:0.0)HTU29:5.0)HTU30:3.0,(Octopus:44.0,(Nautilus:13.0,((Helobdella_robusta:12.0,Scaphopods:6.0)HTU25:46.0,(Chaetopleura_apiculata:16.0,((Aplysia_californica:54.0,Littorina_littorea:34.0)HTU42:38.0,(Yoldia_limatula:11.0,(Lymnaea_stagnalis:6.0,(Perotrochus_lucaya:3.0,(Crassostrea_virginica:16.0,Siphonaria_pectinata:7.0)HTU41:1.0)HTU46:0.0)HTU45:0.0)HTU44:0.0)HTU43:13.0)HTU39:6.0)HTU40:8.0)HTU26:18.0)HTU31:5.0)HTU32:23.0)HTU34:3.0)HTU36:14.0)HTU37:0.0)HTU38:0.0)HTU24[1405.0];
(Antarctic_neomeniomorph:0.0,(Lingula_anatina:0.0,((Cerebratulus_lacteus:54.0,Euprymna_scolopes:20.0)HTU40:10.0,((Venerupis_decussatus:31.0,Venerupis_philippinarum:3.0)HTU38:1.0,(Mytilus_californianus:49.0,((Lottia_gigantea:28.0,(Greenland_neomeniomorph:73.0,(Ilyanassa_obsoleta:0.0,(Nucula:17.0,Solemya:14.0)HTU32:0.0)HTU33:0.0)HTU34:5.0)HTU35:3.0,(Octopus:44.0,(Nautilus:13.0,((Helobdella_robusta:12.0,Scaphopods:6.0)HTU28:46.0,(Chaetopleura_apiculata:16.0,(((Aplysia_californica:54.0,Littorina_littorea:34.0)HTU25:0.0,(Lymnaea_stagnalis:6.0,Yoldia_limatula:11.0)HTU26:0.0)HTU27:10.0,(Perotrochus_lucaya:2.0,(Crassostrea_virginica:15.0,Siphonaria_pectinata:7.0)HTU46:1.0)HTU44:0.0)HTU45:13.0)HTU29:6.0)HTU30:8.0)HTU31:18.0)HTU36:5.0)HTU37:23.0)HTU39:3.0)HTU41:14.0)HTU42:0.0)HTU43:0.0)HTU24[1405.0];
